tests/by-util/test_cksum.rs

```diff
@@ -291,6 +325,20 @@ fn test_length_is_zero() {
         .stdout_is_fixture("length_is_zero.expected");
 }
 
+#[test]
+fn test_length_repeated() {
+    new_ucmd!()
+        .arg("--length=10")
+        .arg("--length=123456")
+        .arg("--length=0")
```

Comment:
I don't know if you have chosen the invalid values `10` and `123456` on purpose. At least they helped me discover a consistency bug in GNU `cksum`: `--length=10 --length=0` shows an error whereas `--length=123456 --length=0` doesn't. They fixed it to match our implementation and no longer show an error in such a case. So we can leave this test as it is.

Thanks for the example :)

Reply:
That was not intentional, but I guess it goes to show that I should really run *every single little command* also with the GNU implementation. :D